oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSink.java

```diff
@@ -102,11 +102,6 @@ protected Map<String, String> getDBSpecificArguments() {
       return ImmutableMap.of(OracleConstants.DEFAULT_BATCH_VALUE, String.valueOf(defaultBatchValue));
     }
 
-    @Override
-    public String getTransactionIsolationLevel() {
```

Comment:
This method should not be removed.

Reply:
Otherwise it will use the default implementation which returns `null` always.
https://github.com/data-integrations/database-plugins/blob/f2b0a7c0801b3ceedd9092a208d64b7fa7ffd03c/database-commons/src/main/java/io/cdap/plugin/db/batch/config/AbstractDBSpecificSourceConfig.java#L207

Reply:
fixed